### ISSUE TITLE
libretro core won't start the game Ruina, giving Error: Invalid encoding: ibm-943_P15A-2003

### ISSUE THREAD

**Issue by fatjing** [EasyRPG/Player#2629]
#### Name of the game:
[Ruina](https://www.freem.ne.jp/win/game/2037)

#### Player platform:
Windows 10 21H1
Retroarch 1.9.8
The core is fetched from within Retroarch's internal updater 

#### Describe the issue in detail and how to reproduce it:
![image](https://user-images.githubusercontent.com/1171924/131790847-bd5c3962-3e09-4a37-9476-14f929fef61f.png)

This problem comes up over and over again. You never know whether it will break upon each core updating or retroarch updating.
I wonder if we could find the root cause and fix it once and for all.

Related: #2470 #2488

**Comment by Ghabry** [comment]
ICU 🙈.

I should consider moving the core from mingw to visual studio. This is at least a configuration we officially support :/.

**Comment by Ghabry** [comment]
a wtf-problem:
Directly after building ICU for mingw cross the buildscript is aborted.
I have code to generate the datafile right after but it is never executed o_O

https://git.libretro.com/libretro/easyrpg-libretro/-/jobs/816072#L4944

**Comment by Ghabry** [comment]
tl;dr
I do not want that the users are negatively impacted by this conflict so I simply added Windows libretro builds to our own infrastructure and simply wget the DLL onto their Gitlab buildbot.

(This means you will get a newer continuous build instead of 0.6.2.3, when we release 0.7 I will pin the release again)

-----------------------

Long story: To solve this once and for all I give up here and migrate from mingw to our upstream solution: VS2019+vcpkg.

Though there is one gigantic problem:
The path the Gitlab CI runner of libretro uses is quite long and the vcpkg package buildtrees are also quite long so it exceeds the 260 character legacy path limit of Windows.

The build works once but the 2nd time the gitlab runner attempts to cleanup the build directory it fails because some filenames are too long. Because I have no control about the checkout process one of the libretro admins has to manually login to the Windows server and delete the directory every time.

This Gitlab CI bug is known since 3 years - Gitlab does not care. The libretro lead also does not care, they say "maybe the time has come to finally start building this like a normal core? this does not seem sustainable at all"

This solution is sustainable. We use vcpkg here since 3 years and it works excellent. I have enough of wasting time here, so our own CI solution does now the heavy lifting :blush: 
--------------------

I will close this issue in a day or two.
